nsq/reader.py

```diff
@@ -626,6 +632,10 @@ def set_max_in_flight(self, max_in_flight):
                     self._send_rdy(conn, 0)
             self.total_rdy = 0
         else:
+            for conn in self.conns.values():
```

Comment:
Should we just reset `rdy_timeout` on all conns regardless of `max_in_flight`?

I realize that we only need it when `max_in_flight > 0` because we reset `rdy_timeout` in `_send_rdy`, which is forcibly called on all connections when `max_in_flight == 0`, but that's _mostly_ true for `_redistribute_rdy_state` too. To me, this is a readability thing for our future selves...

Reply:
Sure. Will make that change.